pkg/jwt/ecdsa-sha3.go

```diff
@@ -0,0 +1,35 @@
+package jwt
+
+import (
```

Comment:
Please don't add a new package called `pkg/jwt` move this code directly into `cmd/config/identity/openid/jwt`